README.md

```diff
@@ -1764,6 +1764,7 @@ _See also: [awesome-sysadmin/Code Review](https://github.com/awesome-foss/awesom
 
 _Related: [Software Development - Project Management](#software-development---project-management), [Ticketing](#ticketing)_
 
+- [AppFlowy](https://www.appflowy.io/) - The Open source alternative to Notion. ([Source Code](https://github.com/AppFlowy-IO/AppFlowy)) `AGPLv3 License`
```

Comment:
- please use the description to... describe what the software does
- correct license tag is `AGPL-3.0`
- missing language/platform tag